packages/shipjs-lib/src/lib/config/defaultConfig.js

```diff
@@ -116,6 +116,8 @@ export default {
   slack: {
     default: {
       username: 'Ship.js',
+      // eslint-disable-next-line camelcase
+      icon_emoji: ':ship:',
```

Comment:
```suggestion
      icon_emoji: ':passenger_ship:',
```

This is the one we're using on our README :)